recipes/fastqvalidator/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export CFLAGS="-I${PREFIX}/include -I${SRC_DIR}/libStatGen/include"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPATH=${PREFIX}/include:${SRC_DIR}/libStatGen/include
+
+# paths for fastqvalidator
+export LIB_PATH_GENERAL=${SRC_DIR}/libStatGen/
+export LIB_PATH_FASTQ_VALIDATOR=${SRC_DIR}/libStatGen/
+
+# compile libstatgen
+cd libStatGen
+make
+cp libStatGen*.a ${PREFIX}/lib
```

Comment:
This line is a problem. It will prevent this package from being installed at the same time as libStatGen. Polluting `/lib` with libraries from other packages is a bad idea...